src/App.tsx

```diff
@@ -1,20 +1,47 @@
-import React, { useReducer } from "react";
+import React, { useEffect, useReducer } from "react";
 import "./App.css";
 import { State } from "./Types/gamestate";
 import { reducer } from "./Reducer/reducer";
 function getInitialState(): State {
-  return { phase: "pre-game" };
+  return {
+    phase: "pre-game",
+    wordPack: null,
+  };
 }
 
 function App() {
   const [state, dispatch] = useReducer(reducer, null, getInitialState);
 
+  useEffect(() => {
+    fetch("fruits.txt")
+      .then((response) => response.text())
+      .then((text) => {
+        setTimeout(
+          () =>
+            dispatch({
+              type: "load-data",
+              wordPack: text
+                .split("\n")
+                .map((word) => word.toUpperCase().trim())
+                .filter(Boolean),
+            }),
+          3000
+        );
+      });
+  }, [dispatch]);
```

Comment:
The dispatch function returned by useReducer doesn't change identity, and I believe the exhaustive dependencies lint rule knows this! If we move to custom hooks, as I'll recommend we do later, then we may need to specify `dispatch` as a dependency to please the lint rule, but for now, we don't!

```suggestion
  }, []);
```